library/psa_crypto.c

```diff
@@ -3719,6 +3719,13 @@ static psa_status_t psa_aead_check_nonce_length( psa_algorithm_t alg,
     return( PSA_ERROR_INVALID_ARGUMENT );
 }
 
+static psa_status_t psa_aead_initial_checks( psa_algorithm_t alg ) {
```

Comment:
Non-blocker: this is specifically a check on the algorithm (as opposed to e.g. checking that the key/nonce size/… is compatible with the algorithm), so a better name would be `psa_aead_check_algorithm`.